hw/drivers/trng/trng_kinetis/src/trng_kinetis.c

```diff
@@ -18,7 +18,23 @@
  */
 
 #include <string.h>
+#include "os/mynewt.h"
+
+#if MYNEWT_VAL(KINETIS_TRNG_USE_RNGA)
+#define USE_RNGA 1
 #include "fsl_rnga.h"
+#elif MYNEWT_VAL(KINETIS_TRNG_USE_TRNG)
+#define USE_TRNG 1
+#include "fsl_trng.h"
+#define TRNG_START(base) \
```

Comment:
maybe use do {} while (0) to prevent accidents with if (cond) TRNG_START();
though I know mynewt style calls for {} in each if

Reply:
Done.